web/src/message_events.ts

```diff
@@ -220,6 +221,7 @@ export let update_views_filtered_on_message_property = (
                     // apply the filter locally and the fetched message
                     // can be used to update other message lists and
                     // cached message data structures as well.
+                    allow_empty_topic_name: true,
```

Comment:
This should probably be not placed just after a comment describing why we didn't have other parameters here.

Reply:
Moved it above the comments.